src/main/java/seedu/address/logic/commands/AddAppointmentCommand.java

```diff
@@ -0,0 +1,70 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DATE;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Appointment;
+import seedu.address.model.person.Person;
+
+/**
+ * Adds an appointment to a person in the patient list.
+ */
+public class AddAppointmentCommand extends Command {
+
+    public static final String COMMAND_WORD = "appt";
```

Comment:
Perhaps the command word can be `addA`, or something similar, so that it is consistent with the format of the other command words?

Reply:
I was considering that as well, but I felt that addA may be a bit unintuitive as compared to records since it's just 1 function for the appointment, but I'm open to changing it as well. Maybe we'll see what the rest think since this is quite minor and easily changeable.